frontend/src/lib/hooks/useAddressBook/utils/isValidAddress.ts

```diff
@@ -0,0 +1,6 @@
+import WAValidator from 'multicoin-address-validator';
```

Comment:
I don't think we need this , because we are using walletcore , and there is `AddressService` , there is `validateAddress` method in it